brayns/common/ImageManager.cpp

```diff
@@ -32,36 +32,36 @@ Texture2DPtr ImageManager::importTextureFromFile(
     if (auto temporary = FreeImage_Load(FreeImage_GetFileType(filename.c_str()),
                                         filename.c_str()))
     {
-        image.reset(FreeImage_IsTransparent(temporary)
-                        ? FreeImage_ConvertTo32Bits(temporary)
-                        : FreeImage_ConvertTo24Bits(temporary));
+        image.reset(FreeImage_ConvertTo32Bits(temporary));
         FreeImage_Unload(temporary);
     }
     else
         return nullptr;
 
-    FreeImage_FlipVertical(image.get());
 #if FREEIMAGE_COLORORDER == FREEIMAGE_COLORORDER_BGR
     freeimage::SwapRedBlue32(image.get());
 #endif
 
-    Texture2DPtr texture(new Texture2D);
+    const auto width = FreeImage_GetWidth(image.get());
+    const auto height = FreeImage_GetHeight(image.get());
+    const auto pitch = FreeImage_GetPitch(image.get());
+    const auto channels = 4;
+
+    std::vector<unsigned char> rawData(height * pitch);
```

Comment:
Is this correct? Shouldn't it be `height * pitch * channels`?

Reply:
Yes it is correct

Reply:
In fact I was not talking about the bytes per Pixel, but the number of channels. But if you tell me that pitch is in bytes and not number of elements, that would explain why this code works (and would always do regardless of channels and bytes per pixel).

Reply:
That's what apparently the pitch in FreeImage is, yes.